packages/logging-bunyan/test/index.js

```diff
@@ -285,9 +284,9 @@ describe('logging-bunyan', function() {
         getWriterProjectId: function() { return 'project1'; }
       };
       const recordWithoutTrace = extend({}, RECORD);
-      const recordWithTrace = extend({
-        'logging.googleapis.com/trace': 'projects/project1/traces/trace1'
-      }, RECORD);
+      var recordWithTrace = extend({}, RECORD);
```

Comment:
nit: [Use a const unless the variable needs to be reassigned](https://google.github.io/styleguide/jsguide.html#features-use-const-and-let). In other languages `const` might connote immutability, but that's not the case for JS (AFAIK).

Reply:
Done.